src/data/hi.yml

```diff
@@ -441,7 +441,6 @@ libraries:
   vida: "विडा एक साधारण पुस्तकालय है जो कैमरा (या वीडियो) आधारित गति पहचान और ब्लॉब ट्रैकिंग कार्यक्षमता को p5js में जोड़ता है। "
   p5.voronoi: "p5.voronoi आपके p5.js रेखाचित्रों में वोरोनोई आरेख बनाने और उनका उपयोग करने के लिए उपकरणों का एक सेट प्रदान करता है।" 
   p5.3D: "WebGL में 3D पाठ और छवियां। "
-  p5.xr: "पी5 के साथ वीआर और एआर स्केच बनाने के लिए एक पुस्तकालय।"
```

Comment:
write P5 instead of its hindi translation for p5.xr

Reply:
ok I have changed that line